website/src/app/kb/readme.mdx

```diff
@@ -29,9 +29,8 @@ Use the links below to get started.
 - [Administer](/kb/administer): Everything related to typical, day-to-day
   management of your deployment.
 - [Automate](/kb/automate): Deploy with automation tools like Terraform.
-- [End-user guides](/kb/user-guides): Instructions you can pass along to
-  Firezone end-users, such as installing Clients, using them, and related
-  troubleshooting tips.
+- [Client apps](/kb/client-apps): Documentation for installing and using the
```

Comment:
Looks like this one got skipped, so I updated it

Reply:
Good catch!